osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/services/ResendNotificationService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.core.application.services;
+
+import javax.annotation.PostConstruct;
+import org.opensmartgridplatform.adapter.ws.schema.core.notification.NotificationType;
+import org.opensmartgridplatform.adapter.ws.shared.services.AbstractResendNotificationService;
+import org.opensmartgridplatform.adapter.ws.shared.services.NotificationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service(value = "resendNotificationServiceCore")
+@Transactional(value = "transactionManager")
+public class ResendNotificationService extends AbstractResendNotificationService<NotificationType> {
+
+  @Autowired private NotificationService smartMeteringNotificationService;
+
+  @Value("${web.service.notification.application.name}")
```

Comment:
If @Value annotations are scattered all over the code, it will be harder to keep track of all the properties in use.

As another option, would it make sense to move this property within the `NotificationService`? 

I've notices this property is already used in @Value annotations in several places, thus for now I'll guess it is OK to leave it as is, but as stated before, it is not my preferred choice.